packages/sui-precommit/package.json

```diff
@@ -13,11 +13,13 @@
   "author": "",
```

Comment:
Avoid sui-precommit installing a specific linter version and instead rely on the one installed on the project.